src/mongoc/mongoc-server-description-private.h

```diff
@@ -104,6 +94,10 @@ mongoc_server_description_set_set_version (mongoc_server_description_t *descript
 void
 mongoc_server_description_set_election_id (mongoc_server_description_t *description,
                                            const bson_oid_t            *election_id);
+
+const char *
+mongoc_server_description_type_string (const mongoc_server_description_t *description);
```

Comment:
Errrrrrrr. Private functions should start with _, but I guess no other function here does that :/